pkg/controller/secret/secret_controller.go

```diff
@@ -152,7 +152,8 @@ func createPagerdutyRoute() *alertmanager.Route {
 		{Receiver: receiverMakeItWarning, Match: map[string]string{"alertname": "KubeAPILatencyHigh", "severity": "critical"}},
 
 		// https://issues.redhat.com/browse/OSD-3086
-		{Receiver: receiverPagerduty, MatchRE: map[string]string{"exported_namespace": alertmanager.PDRegex}},
+		// https://issues.redhat.com/browse/OSD-5872
+		{Receiver: receiverPagerduty, MatchRE: map[string]string{"exported_namespace": alertmanager.PDRegex}, Match: map[string]string{"prometheus": "openshift-monitoring/k8s"}},
```

Comment:
This change only impacts alerts where `exported_namespace` matches our regex.  The `prometheus` label should be checked for EVERYTHING routed to the Receiver `receiverPagerduty`.

Reply:
added the label for all PD subroutes. PTAL @jewzaam